src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
+
+        ball.setBallColor(ColorSupplier.getRandomColor());
+
+        ball.setBallNumber(new Random().nextInt(100));
+
+        return ball;
```

Comment:
```suggestion
        String color = colorSupplier.getRandomColor();
        int number = random.nextInt(MAX_BALL_NUMBER);
        return new Ball(color, number);
```
Don't use default constructor and setters, instead you should create local variables of color and number and use them in parameterized constructor of Ball class.
Use colorSupplier (field of Lottery class) on object level access (not class)
Use random (field of Lottery class) on object level access (not create new Random() object every execution time of this method). 
Don't  use magic numbers! What is 100? You should define MAX_BALL_NUMBER variable on class level.
Delete redundant empty lines.